tests/integration/data/datasets.py

```diff
@@ -168,3 +188,27 @@ def org_annotate_model_group(indico, org_annotate_dataset: Dataset, org_annotate
     mg = client.call(GetModelGroup(id=component.model_group.id, wait=True))
 
     return mg
+
+@pytest.fixture(scope="module")
+def org_annotate_exchange_integration(org_annotate_workflow: Workflow) -> Integration:
+    client = IndicoClient()
+    creds = {
+        "clientId": os.getenv("EXCH_CLIENT_ID"),
```

Comment:
are there some repo docs somewhere that specify that this env var should be set? there's also the pytest [skipif option](https://docs.pytest.org/en/7.1.x/how-to/skipping.html#id1) which will check a conditional before deciding if the test should run. so for example skipif(os.getenv(CLIENT_ID) is None)

Reply:
> are there some repo docs somewhere that specify that this env var should be set?

not yet. the integration tests aren't in great shape-- we don't have any documentation for them, and they're not run during CI/CD because a significant portion time out. @goatrocks perhaps we should think about cleaning up and documenting these integration tests in the near future?